lib/assert.ts

```diff
@@ -1,10 +1,8 @@
-import { Error as ErrorCtor } from './primordials';
-
 export function assert(condition: boolean, message: string): asserts condition {
-  if (!condition) throw new ErrorCtor(`assertion failure: ${message}`);
+  if (!condition) throw new Error(`assertion failure: ${message}`);
```

Comment:
OMG the changes in this commit make the library so much easier to read! Thanks for doing this.